common/tests/test_metrics.py

```
@@ -0,0 +1,309 @@
+    def test_exception_propagates(self):
+        with self.assertRaises(KeyError):
+            with metrics.time_random_insult_stage("serialization"):
+                raise KeyError("missing")
+
+
+# ---------------------------------------------------------------------------
+# sql_instrumentation
+# ---------------------------------------------------------------------------
+
+class SqlInstrumentationTests(TestCase):
+    databases = ["default"]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)